pkgs/applications/audio/eartag/default.nix

```
@@ -0,0 +1,76 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook4
+, libadwaita
+, gettext
+, glib
+, desktop-file-utils
+, appstream-glib
+, gtk4
+, librsvg
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "eartag";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "knuxify";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-vNAKGwcvkZ3YdrA0uYTnlyta2az/bmjgCE/6e9NADT0=";
+  };
+
+  postPatch = ''
+    patchShebangs ./build-aux/meson/postinstall.py
+    chmod +x ./build-aux/meson/postinstall.py
+    substituteInPlace ./build-aux/meson/postinstall.py \
+      --replace "gtk-update-icon-cache" "gtk4-update-icon-cache"
+  '';
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    glib
+    desktop-file-utils
+    appstream-glib
+    pkg-config
+    gettext
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    librsvg
+    libadwaita
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pygobject3
+    eyeD3
+    pillow
+    mutagen
+    pytaglib
+    python_magic
```

`error: undefined variable 'python_magic'`

---

This is certainly strange, I don't get this error. `python3Packages` is pointed to `python310Packages` and it has that package available.

---

No, this is not strange. python_magic is an alias which is forbidden in nixpkgs but not on your local config.

```suggestion
    python-magic
```

---

Ah, got it.